Sources/Dependencies/Dependency.swift

```diff
@@ -233,19 +243,19 @@ extension DependencyValues {
     get {
       self[
         Key.self,
-        key.fileID,
-        key.filePath,
-        key.line,
-        key.column
+        fileID: key.fileID,
+        filePath: key.filePath,
+        line: key.line,
+        column: key.column
```

Comment:
Noticed the lack of labels when fixing docs. This is technically breaking but I don't think anyone should be invoking these arguments directly outside the library and there's a fix-it to add the labels, but if we're worried I can undo this.